### PR TITLE
doc update - darwin build

### PR DESCRIPTION
**Pull request by navidys**
updating README.md and install.md to include darwin build. 

Signed-off-by: Navid Yaghoobi <navidys@fedoraproject.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ It is using [podman go bindings](https://github.com/containers/podman/tree/main/
 
 ## Installation
 
-Building from source (Linux, Windows) or installing packaged versions are detailed in [install guide](install.md).
+Building from source (Linux, Windows and Darwin) or installing packaged versions are detailed in [install guide](install.md).
```
Comment (rhatdan) [inline README.md:24]: You sure you want to call this Darwin, or just Mac?

Comment (navidys) [reply README.md:24]: You are right, I think it is better to call it Mac, will update the PR and change it to Mac.

**Comment by navidys** [comment]
@rhatdan I have updated README.md and install.md and replaced "darwin" with "MacOS". 

Shall I update the Makefile and change "binary-darwin" target  to "binary-mac" ?

File: install.md [hunk 2]
```diff
@@ -2,36 +2,47 @@
 
 - [**Building From Source**](#building-from-source)
 - [**Installing Packaged Versions**](#installing-packaged-versions)
-  - [**Arch Linux (AUR)**](#arch-linux-aur)
   - [**Fedora**](#fedora)
+  - [**Arch Linux (AUR)**](#arch-linux-aur)
```
[the remark this answers is not in the document]
Comment (navidys) [reply install.md:6]: My mistake, I was going to add two different sections:
1. offical repository (fedora, ..)
2. user repository (aur, copr and snap) 

will revert it back,  thanks for letting me know.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**